api/DocumentTimeline.json

```diff
@@ -68,10 +68,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": false
+              "version_added": "60"
```

Comment:
According to the bug, this landed in Firefox 50.

Reply:
You're so smart ;-)

Dammnit, don't know how I missed that. Updated now!